renku/service/jobs/cleanup.py

```diff
@@ -21,61 +21,107 @@
 import time
 from datetime import datetime
 
-from renku.service.cache import FileManagementCache, ProjectManagementCache
+from renku.service.cache import ServiceCache
+from renku.service.cache.models.job import USER_JOB_STATE_ENQUEUED, \
+    USER_JOB_STATE_IN_PROGRESS
+
+
+def file_cleanup_lock_check(jobs, file):
+    """Check if file should be deleted."""
+    for job in jobs:
+        if file.file_id in job.locked:
+            return True
+
+    return False
+
+
+def file_cleanup_ttl_check(file, ttl=None):
+    """Check if file should be deleted."""
+    ttl = ttl or int(os.getenv('RENKU_SVC_CLEANUP_TTL_FILES', 1800))
+
+    # If record does not contain created_at,
+    # it means its an old record, and we should mark it for deletion.
+    created_at = (file.created_at -
+                  datetime.utcfromtimestamp(0)).total_seconds() * 1e+3
+
+    if not file.created_at:
+        created_at = (time.time() - ttl) * 1e+3
```

Comment:
Shouldn't you just return here directly? because `time.time() - (time.time() - ttl)` is guaranteed to be `> ttl` further down, so this code makes it look like some calculation is happening, even though it just returns.

Reply:
Right, this look's like my rebase went bad. This is not up to date code, let me check if last commit is pushed. Thanks for the review btw!